packages/dashboards/src/lib/configurator/components/widgets/table/columns-editor-v2/table-columns-configuration-v2.component.html

```diff
@@ -0,0 +1,85 @@
+<div class="nui-table-columns-configuration">
+    <div *ngIf="!dataSourceError && dataSourceFields.length" cdkDropList (cdkDropListDropped)="drop($event)"
+         [formGroup]="form">
+        <div formArrayName="columns">
+            <div cdkDrag
+                 (cdkDragStarted)="cdkDragStarted($event)"
+                 class="nui-table-columns-configuration__node-item"
+                 *ngFor="let columnForm of columnForms; index as i; trackBy: trackBy">
+
+                <nui-drop-area class="nui-table-columns-configuration__drop-area" *cdkDragPlaceholder [style.height.px]="draggedItemHeight"></nui-drop-area>
+
+                <ng-template #headerText>
+                    <ng-container>
+                        <div class="nui-table-columns-configuration__drag-handler" cdkDragHandle>
+                            <nui-icon icon="drag"></nui-icon>
+                        </div>
+                    </ng-container>
+                    <span class="nui-table-columns-configuration__header-text">
+                    {{ columnForm?.value.label | nuiWidgetConfiguratorSectionHeader : i : "Column" }} <!-- TODO: i18n -->
```

Comment:
is the i18n going to be done as part of this task? if a future task, is there a ticket?